docs/toc.js

```diff
@@ -169,9 +169,21 @@ module.exports = {
           type: 'link',
         },
         {
-          pathSegment: 'importing-stories-in-tests',
```

Comment:
Because a path was renamed, please remember to open a PR adding a redirect in `frontpage`.

Reply:
I haven't forgotten about it. I wanted to get this vetted first and then update the redirect.